proto/replicationdata.proto

```diff
@@ -66,7 +66,6 @@ message Configuration {
 message StopReplicationStatus {
   replicationdata.Status before = 1;
   replicationdata.Status after = 2;
-  bool backup_running = 3;
```

Comment:
A field for this is already present in replicationdata.Status, no need to add another one here which just led to confusion in the previous PR.

Reply:
This would be a backwards compatibility breaking change. I don't know if we can safely remove it, but if we can, the first step would be to explicitly mark it as deprecated before it can be removed. We can't in general just delete proto fields without any explicit deprecation process.

Reply:
The original PR that added this field has not gone into any release, so it would be ok to simply delete the field.

Reply:
Ah, that's good then!

Reply:
This field was just added a couple of weeks ago and has not made it into any release yet. So I think that removing it is safe, but can also leave it in place marking it as deprecated if you prefer.